### PR TITLE
Add copy for Top Scorers concept exercise, automated analysis, for Elm track

### PR DESCRIPTION
**Pull request by ceddlyburge**
Comments for Elm's top-scorers concept exercise

[Sister PR in elm-analyzer](https://github.com/exercism/elm-analyzer/pull/36)

@jiegillet 
@mpizenberg

### CODE REVIEW

File: analyzer-comments/elm/top-scorers/use_filter.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# use filter
+
+`resetPlayerGoalCount` is most easily written using `Dict.insert`, and using it is also a learning goal for the exercise.
```
Comment (jiegillet) [inline analyzer-comments/elm/top-scorers/use_filter.md:3]: I think `use_filter` and `use_insert` were swapped.

File: analyzer-comments/elm/top-scorers/use_foldl.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# use foldl and updateGoalCountForPlayer
+
+`aggregateScorers` is most easily written using `List.fold1` and `updateGoalCountForPlayer`.
```
Comment (jiegillet) [inline analyzer-comments/elm/top-scorers/use_foldl.md:3]:
```suggestion
`aggregateScorers` is most easily written using `List.foldl` and `updateGoalCountForPlayer`.
```

File: analyzer-comments/elm/top-scorers/use_foldl.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# use foldl and updateGoalCountForPlayer
```
Comment (jiegillet) [inline analyzer-comments/elm/top-scorers/use_foldl.md:1]: If this comment identical to `use_updateGoalCountForPlayer.md`, then I think it's wiser to only have one, and change the rule to be 

```elm
aggregateScorersMustUseFoldlAndUpdateGoalCountForPlayer : Rule
aggregateScorersMustUseFoldlAndUpdateGoalCountForPlayer =
    Analyzer.functionCalls
        { calledFrom = TopFunction "aggregateScorers"
        , findFunctions = [ FromExternalModule [ "List" ] "foldl", FromSameModule "updateGoalCountForPlayer"  ]
        , find = All
        , comment = Comment "Doesn't use List.foldl or updateGoalCountForPlayer" "elm.top-scorers.use_foldl" Essential Dict.empty
        }
```

File: analyzer-comments/elm/top-scorers/use_withDefault.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# use map
```
Comment (jiegillet) [inline analyzer-comments/elm/top-scorers/use_withDefault.md:1]: Same thing here with `use_map`, this could be a single rule.

**Comment by ceddlyburge** [comment]
Hi @jiegillet , I think I've resolved those comments now. Cheers, Cedd

**Review by jiegillet — APPROVED** [review]
Awesome!

**Pull request merged**